CMakeLists.txt

```diff
@@ -78,19 +78,19 @@ set(INSTALL_GMOCK OFF)
 set(INSTALL_GTEST OFF)
 set(BENCHMARK_ENABLE_INSTALL OFF)
 
-find_package(Git QUIET)
+# find_package(Git QUIET)
```

Comment:
Why keep this as a comment if not needed?

Reply:
This was actually an oversight, I commented this out for debugging purposes.